source/guide/3.2/guide-policies.md

```diff
@@ -106,33 +183,24 @@ Built-in policies are not special in any way - they use the same API any other c
 Advanced users may wish to write custom policies.
 To learn how to write a custom policy, refer to the [policies authoring guide](guide-authoring-policies.html).
 
-<!---
 # Auto Healing
 
 Auto healing is the process of automatically identifying when a certain component of your system is failing and fixing the system state without any user intervention.
 
-Cloudify comes with built-in support for auto healing different components.
+Cloudify comes with a built-in support for auto healing different components.
 
-There are several things that need to be configured to work with auto healing. This guide will walk you through them, and explain the limitations the mechanism currently has.
+There are several things that need to be configured for auto healing to work. This guide will walk you through them and explain the mechanism's current limitations.
 
 In short, the process involves:
 
-* Configuring monitoring on compute nodes that should have auto healing.
+* Configuring monitoring on compute nodes that will be subject to autohealing.
 * Adding the `autoheal` workflow to the blueprint.
```

Comment:
This section does not reflect that fact that heal is now a built-in workflow and should not be manually included by the user